iOS_SDK/OneSignalSDK/Source/OSSubscription.m

```diff
@@ -61,10 +61,10 @@ - (BOOL)compare:(OSSubscriptionState*)from {
 
 - (instancetype)initAsFrom {
     let standardUserDefaults = OneSignalUserDefaults.initStandard;
-    _accpeted = [standardUserDefaults getSavedBoolForKey:ACCEPTED_PERMISSION defaultValue:false];
-    _userId = [standardUserDefaults getSavedStringForKey:USERID_LAST defaultValue:nil];
-    _pushToken = [standardUserDefaults getSavedStringForKey:PUSH_TOKEN defaultValue:nil];
-    _userSubscriptionSetting = [standardUserDefaults getSavedObjectForKey:SUBSCRIPTION_SETTING defaultValue:nil] == nil;
+    _accpeted = [standardUserDefaults getSavedBoolForKey:OSUD_ACCEPTED_PERMISSION defaultValue:false];
```

Comment:
Should be ` OSUD_ACCEPTED_PERMISSION_LAST`

Reply:
Got it!